pkg/controllers/mpi_job_controller.go

```diff
@@ -940,18 +934,26 @@ func newLauncherRole(mpiJob *kubeflow.MPIJob, workerReplicas int) *rbacv1.Role {
 	}
 }
 
+func getLaunchName(mpiJob *kubeflow.MPIJob) string {
+	return mpiJob.Name + launcherSuffix
+}
+
+func getLabelMap(mpiJob *kubeflow.MPIJob) map[string]string {
```

Comment:
Consider renaming to `getLabelsMap` since we are using this for `Labels` and `MatchLabels` fields.